iterators.tex

```diff
@@ -3591,6 +3644,43 @@
 \tcode{return x + n;}
 \end{itemdescr}
 
+\indexlibrary{\idxcode{iter_move}!\idxcode{move_iterator}}%
```

Comment:
We have two styles of presentation for iterator adapters: subclause-per-function, and subclause-per-grouping. This subclause is a grouping of "non-member functions", so `iter_move` and `iter_swap` fit right in here without a distinct subsection heading.